pages/auth/logout.tsx

```diff
@@ -1,22 +1,27 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { useTranslation } from "next-i18next";
 import Link from "next/link";
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
 import { GetServerSideProps } from "next";
-import { useSession } from "next-auth/react";
 
 const Logout = () => {
   const { i18n, t } = useTranslation("logout");
-  const { data: session } = useSession();
+  const [lastLoginTime, setLastLoginTime] = useState("");
+
+  useEffect(() => {
```

Comment:
`useEffect` was used to circumvent a [react-hydration-error](https://nextjs.org/docs/messages/react-hydration-error) because of the API being used to retrieve client's timezone.